src/panels/history/ha-panel-history.ts

```diff
@@ -101,7 +101,7 @@ class HaPanelHistory extends LitElement {
                   .hass=${this.hass}
                   .historyData=${this._stateHistory}
                   .endTime=${this._endDate}
-                  no-single
+                  noSingle
```

Comment:
Better to make this none breaking and keep the attr `no-single`.

Change:
```
  @property() public noSingle = false;
```
Too:
```
  @property({type: Boolean, attribute: "no-single"}) public noSingle = false;
```

Reply:
The risk you see with breaking is coming from custom cards using that element? Inside this repo there are no uses anymore of "no-single".

Reply:
yeah